guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -504,9 +506,13 @@ class View extends Template
 
 6.1.1. Application SHOULD be structured in compliance with the [CQRS principle].
 
-6.1.2. Every application layer (Presentation, Service Contracts, Data Access) MUST process (handle or re-throw) exceptions of the underlying layer.
+6.1.2. Every application layer (Presentation, Service Contracts, Data Access) 
+    MUST process (handle or re-throw) exceptions of the underlying layer.
 
-6.2.5. Configuration for the presentation layer MUST be declared in the corresponding application area. This includes events and plugins that customize the presentation layer.
+6.1.3. A layer MUST NOT depend on layer that invokes it.
+          Example, a service that manages products MUST NOT invoke component 
```

Comment:
Example is not clear.

Should we rephrase 6.1.3 to prohibit dependency of Domain on Presentation, Persitence on the Domain etc. We can also provide a reference to Magento layering diagram.

Reply:
6.1.3. A layer MUST only depend on a layer directly below it.
Add reference to a diagram with description of layers.